### PR TITLE
BZ-1754052: Added info on managing CSRs and additional node cert info

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1754052/install_config/redeploying_certificates.html#cert-expiry-managing-csrs

### CODE REVIEW

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -730,3 +732,69 @@ $ oc delete secret router-metrics-tls
...
+[[cert-expiry-auto-approving-csrs]]
+=== Configuring Automatic Approval of Certificate Signing Requests
+
+You can configure automatic approval of node certificate signing requests (CSRs) by specifying adding the following parameter to your Ansible inventory file when installing your cluster:
+
+----
+openshift_master_bootstrap_auto_approve=true
+----
+
+This allows all CSRs generated using the boostrap credential or from a previously authenticated node with the same host name to be approved without any administrator intervention.
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc]: I'd s/This/Adding this parameter

s/using/by using

**Review by kalexand-rh — COMMENTED** [review]
Just a style pick. It LGTM.

**Comment by bergerhoffer** [comment]
@rjhowe Can you review this please? Thanks!

**Comment by rjhowe** [comment]
@bergerhoffer  Looks good.

**Comment by bergerhoffer** [comment]
Hi @gpei - can you please review this update? Thanks!

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1754052/install_config/redeploying_certificates.html#cert-expiry-managing-csrs

**Comment by bergerhoffer** [comment]
Or @barleyer - I'm not sure who is best to review this.

**Comment by gpei** [comment]
LGTM

**Pull request merged**